metricflow/sql/sql_plan.py

```diff
@@ -42,19 +42,19 @@ def parent_nodes(self) -> List[SqlQueryPlanNode]:  # noqa: D
     @abstractmethod
     def accept(self, visitor: SqlQueryPlanNodeVisitor[VisitorOutputT]) -> VisitorOutputT:
         """Called when a visitor needs to visit this node."""
-        pass
+        raise NotImplementedError
```

Comment:
This shouldn't matter, but I like the consistency.

Reply:
Yeah, it seems like that's what the docs suggest, though I've seen some conflicting examples.